test/core/card.spec.js

```diff
@@ -0,0 +1,22 @@
+const expect = require('./../helper').expect;
+
+const Card = require('./../../src/core/card');
+
+describe('Card', () => {
+  beforeEach(() => {
+    this.card = new Card(2, 'spades');
+  });
+
+  describe('#constructor', () => {
+    it('initializes a new card', () => {
+      expect(this.card.value).to.equal(2);
+      expect(this.card.seed).to.equal('spades');
```

Comment:
For what I see here the test consists in checking that the card has always value = 2 and seed = spades. Instantiate the class in the `it` block to make it more clear. 

Also you shouldn't use always use the same instance, you should continuously change the value of the constructor to make the tests more generic. See [triangulation](http://elnur.pro/triangulation-in-testing)